.gitpod.yml

```
@@ -35,6 +35,7 @@ tasks:
       export API_LOCATION=$(gp url 3000)
       export CHALLENGE_EDITOR_API_LOCATION=$(gp url 3200)
       export CHALLENGE_EDITOR_CLIENT_LOCATION=$(gp url 3300)
+      CHALLENGE_EDITOR_LEARN_CLIENT_LOCATION=$(gp url 8000)
```

```suggestion
      export CHALLENGE_EDITOR_LEARN_CLIENT_LOCATION=$(gp url 8000)
```
The export is required, otherwise the variable only exists for the duration of the script: https://superuser.com/a/47897

If you want to check it on Gitpod, you just need to open a terminal (Ctrl-Shift-C, on Linux at least) and run the challenge editor as you would normally.